### DISCUSSION TITLE
Minification of the magento original javascript files

### DISCUSSION THREAD

**Discussion by bhennesAdv** [OpenMage/magento-lts#2107] [Ideas]
### Description (*)
More and more trending subject in the web domain are about optimization of the data consumed by the users when browsing websites.
Javascript files in OpenMage (ROOT/js/* or the default skin ROOT/skin/frontend/*/*/js/) could be minified by default (Source must still be present in the repository for evolution/correction purpose).

### Expected behavior (*)
The default js files loaded in the OpenMage pages should all be minified.

### Benefits
Several kb for each user using an OpenMage website and no need for OpenMage developpers to add a minifying process in the website.

### Additional information
As I said in the title, this is micro-optimization. The aim is not to add any dependency with a JavaScript minifiying process but to provide the default files properly minified (With licenses comment properly kept).

What do you think about it ?
Regards,

**Comment by fballiano** [comment]
I prefer using a separate module for minimization or manage it separately (eg: by the CDN) while I would prefer to have the source code unminified

**Comment by loekvangool** [comment]
Agree with @fballiano. At least on Cloudflare CDN, this feature is available even on the free plan.
